engines/scumm/gfx_gui.cpp

```diff
@@ -2559,6 +2748,217 @@ void ScummEngine::setUpMainMenuControls() {
+	}
+
+	// Outer box
+	setUpInternalGUIControl(GUI_CTRL_OUTER_BOX,
+							getBannerColor(4),
+							getBannerColor(2),
+							getBannerColor(13),
+							getBannerColor(14),
+							getBannerColor(15),
+							getBannerColor(16),
+							getBannerColor(6),
+							getBannerColor(4),
+							20,
+							yConstant - 60,
+							300,
+							((yConstant + 60) < 0 ? -120 : yConstant + 60),
+							_emptyMsg, 1, 1);
+
+	// Load button
+	setUpInternalGUIControl(GUI_CTRL_LOAD_BUTTON,
+							getBannerColor(4),
+							getBannerColor(5),
+							getBannerColor(17),
+							getBannerColor(18),
+							getBannerColor(19),
+							getBannerColor(20),
+							getBannerColor(6),
+							getBannerColor(7),
+							isJap ? 26 : 44,
+							yConstant - 31,
+							isJap ? 111 : 103,
+							yConstant - 31 + 22,
+							getGUIString(gsLoad), 1, 1);
+
+	// Play button
+	setUpInternalGUIControl(GUI_CTRL_PLAY_BUTTON,
+							getBannerColor(4),
+							getBannerColor(5),
+							getBannerColor(17),
+							getBannerColor(18),
+							getBannerColor(19),
+							getBannerColor(20),
+							getBannerColor(6),
+							getBannerColor(7),
+							isJap ? 26 : 44,
+							yConstant - 5,
+							isJap ? 111 : 103,
+							yConstant - 5 + 22,
+							getGUIString(gsPlay), 1, 1);
+
+	// Restart button
+	setUpInternalGUIControl(GUI_CTRL_RESTART_BUTTON,
+							getBannerColor(4),
+							getBannerColor(5),
+							getBannerColor(17),
+							getBannerColor(18),
+							getBannerColor(19),
+							getBannerColor(20),
+							getBannerColor(6),
+							getBannerColor(7),
+							isJap ? 26 : 44,
+							yConstant + 21,
+							isJap ? 111 : 103,
+							yConstant + 21 + 22,
+							getGUIString(gsRestart), 1, 1);
+
+	if (_menuPage == GUI_PAGE_MAIN) {
+		// Arrow left button
+		setUpInternalGUIControl(GUI_CTRL_ARROW_LEFT_BUTTON,
+								getBannerColor(4),
+								getBannerColor(5),
+								getBannerColor(17),
+								getBannerColor(18),
+								getBannerColor(19),
+								getBannerColor(20),
+								getBannerColor(6),
+								getBannerColor(7),
+								235,
+								yConstant + 18,
+								247,
+								yConstant + 34,
+								_arrowLeft, 1, 1);
+
+		// Arrow right button
+		setUpInternalGUIControl(GUI_CTRL_ARROW_RIGHT_BUTTON,
+								getBannerColor(4),
+								getBannerColor(5),
+								getBannerColor(17),
+								getBannerColor(18),
+								getBannerColor(19),
+								getBannerColor(20),
+								getBannerColor(6),
+								getBannerColor(7),
+								248,
+								yConstant + 18,
+								260,
+								yConstant + 34,
+								_arrowRight, 1, 1);
+
+		setUpInternalGUIControl(GUI_CTRL_TEXT_SPEED_SLIDER,
+								getBannerColor(9),
+								getBannerColor(10),
+								getBannerColor(18),
+								getBannerColor(17),
+								getBannerColor(20),
+								getBannerColor(19),
+								getBannerColor(10),
+								getBannerColor(12),
+								152,
+								yConstant + 18,
+								235,
+								yConstant + 34,
+								_uncheckedBox, 1, 1);
+	}
```

Comment:
Maybe add `return;` here?

Reply:
Whoops! Yes, given the purpose of the code I'd say that changing the other `if`s to `else if` would improve readability and underline the fact that the menu pages are mutually exclusive to each other.